modules/local/openms_decoydatabase.nf

```diff
@@ -8,29 +8,33 @@ process OPENMS_DECOYDATABASE {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::openms=2.5.0" : null)
+    conda (params.enable_conda ? "bioconda::openms=2.6.0" : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/openms:2.5.0--h4afb90d_6"
+        container "https://depot.galaxyproject.org/singularity/openms:2.6.0--h4afb90d_0"
     } else {
-        container "quay.io/biocontainers/openms:2.5.0--h4afb90d_6"
+        container "quay.io/biocontainers/openms:2.6.0--h4afb90d_0"
     }
 
     input:
         tuple val(meta), path(fasta)
 
     output:
         tuple val(meta), path("*_decoy.fasta"), emit: decoy
-        path "*.version.txt", emit: version
+        path "versions.yml", emit: versions
 
     script:
         def software = getSoftwareName(task.process)
         def prefix   = options.suffix ? "${fasta.baseName}_${options.suffix}" : "${fasta.baseName}_decoy"
 
         """
-            DecoyDatabase -in ${fasta} \\
-                -out ${prefix}.fasta \\
-                -decoy_string DECOY_ \\
-                -decoy_string_position prefix
-            echo \$(FileInfo --help 2>&1) | sed 's/^.*Version: //; s/ .*\$//' &> ${software}.version.txt
+        DecoyDatabase -in ${fasta} \\
+            -out ${prefix}.fasta \\
+            -decoy_string DECOY_ \\
+            -decoy_string_position prefix
```

Comment:
Ah here the indentation is as I would expect (aligned with `"""`), sort of makes the other modules inconsistent...